bin/bin

```diff
@@ -0,0 +1 @@
+bin
```

Comment:
No worries, we are happy to receive a PR from new contributors!
Same here, I currently live in Tokyo :)